### DISCUSSION TITLE
Configuring Managers in onboardingConfig

### DISCUSSION THREAD

**Discussion by zzzpoint** [renovatebot/renovate#25451] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.33.1

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

I want to change the file matching configuration for gitlabci manager to update image version in .gitlab-ci.yaml and .devcontainer/devcontainer.json at the same time. It works fine if I update renovate.json directly in the managed project. But if I include it into onboarding config and run renovate-config-validator - it fails, error in the attached logs. Am I doing something wrong or there is a bug in config validator? I also attach my config.js of renovate bot.
[config.zip](https://github.com/renovatebot/renovate/files/13179384/config.zip)

### Relevant debug logs

<details><summary>Logs</summary>

```
EBUG: Using RE2 as regex engine
DEBUG: Checking for config file in /workspaces/renovate/config.js
 INFO: Validating config.js
ERROR: Found errors in configuration
       "file": "config.js",
       "errors": [
         {
           "topic": "Configuration Error",
           "message": "The \"gitlabci\" object can only be configured at the top level of a config but was found inside \"onboardingConfig\""
         }
       ]
 WARN: Found errors in configuration
       "file": "config.js",
       "warnings": [
         {
           "topic": "Config warning",
           "message": "\"fileMatch\" must be configured in a manager block and not here: onboardingConfig.gitlabci"
         }
       ]
```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by rarkins** [comment]
gitlabci manager does not support json

**Reply by zzzpoint** [reply]
It still works though if I add gitlabci fileMatch into renovate.json directly in the managed project. That's not the issue, I can't have gitlabci in onboarding config for some reason.
